geonotebook/wrappers/file_reader.py

```diff
@@ -69,6 +71,11 @@ def __del__(self):
         self.dataset.close()
 
     def index(self, *args, **kwargs):
+        target_srs = self.dataset.crs
+        # Annotations are always in WGS84
```

Comment:
I agree. That is why I added a comment. It could change in the future. As long as we know the annotation projection we can use the utility function for transforming coordinates